src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -165,6 +166,11 @@ public ModifiableRealmIdentity getRealmIdentityForUpdate(final Principal princip
         return principal instanceof NamePrincipal ? getRealmIdentity(principal.getName(), true) : ModifiableRealmIdentity.NON_EXISTENT;
     }
 
+    @Override
+    public void registerIdentityChangeListener(Consumer<Principal> listener) {
+        // no need to register the listener given that changes to identities are done through the realm
```

Comment:
I think we'll need to implement this method properly at some point even though it doesn't affect the behavior of cache operations.  But it's not necessary for this PR.